frontend/packages/core/src/Resolver/input.tsx

```diff
@@ -55,6 +74,8 @@ const QueryResolver: React.FC<QueryResolverProps> = ({ schemas, submitHandler })
         error={!!error}
         helperText={error?.message || error?.type || ""}
         endAdornment={<SearchIcon />}
+        isAutoCompleteable={isAutoCompleteable}
+        autocompleteCallback={v => autoComplete(inputType, v)}
```

Comment:
Im still looking at how to make optional props so I can remove `isAutoCompleteable`

Reply:
you got it :)

Reply:
Do I need to dynamically assign the `autoComplete` function then? Currently it's always truth right?